### PR TITLE
Feature/news story

### PR DESCRIPTION
**Pull request by elhaw**
###

- separated Article component in  another file so that we can use it later

- created story for News component

### CODE REVIEW

File: src/components/NewsAndArticles/index.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const News = () => {
     );
   }
 
-  if (data) {
+  if (!loading && !error && data) {
```
Comment (AbdoMgdy) [inline src/components/NewsAndArticles/index.jsx:101]: Can you please explain why not just data?

File: src/components/NewsAndArticles/index.jsx [hunk 2]
```diff
@@ -108,12 +108,16 @@ const News = () => {
...
+              style={slideHead}
+              ref={ref}
+              className="md:col-span-5 text-center md:text-left"
+            >
               <Heading
                 primaryText={`${heading_primary}`}
                 secondaryText={`${heading_secondary}`}
                 primaryTextColor="dark"
-                primaryClassName="md:col-span-5 text-center md:text-left"
+                // primaryClassName="md:col-span-5 text-center md:text-left"
```
Comment (Tramsi) [inline src/components/NewsAndArticles/index.jsx:120]: Why commenting this ? the text is now left aligned in mobile view while it should be centered ... i suggest moving only the positioning (md:col-span-5) to the outer div

File: src/components/NewsAndArticles/index.jsx (hunk 1 above)
Comment (elhaw) [reply src/components/NewsAndArticles/index.jsx:101]: 🤔 May be overthinking.
 I thought that maybe after loading directly some error occurs.
so I wanted to make sure 100% that loading is finished and no errors. 

Or I may be wrong.

Comment (Nilegfx) [reply src/components/NewsAndArticles/index.jsx:101]: Over-engineering

Comment (AbdoMgdy) [reply src/components/NewsAndArticles/index.jsx:101]: Yea I think you are overthinking it. it's single threaded so the first 2 conditions will always be checked first. you can leave it if you want i just think it's not necessary

File: src/components/NewsAndArticles/index.jsx (hunk 2 above)
Comment (elhaw) [reply src/components/NewsAndArticles/index.jsx:120]: Actually I did check styling. I will have a look at it

**Pull request merged**